portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,11 +54,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       Translate translate = TranslateOptions.getDefaultInstance().getService();
       Translation translation = translate.translate(commentText, Translate.TranslateOption.targetLanguage(languageCode));
       String translatedText = translation.getTranslatedText();
-      Comment comment = new Comment(translatedText, timestamp, null, id);
+      translatedText = translatedText.replace("&#39;", "'");
+      Comment comment = new Comment(translatedText, timestamp, id);
       comments.add(comment);
     }
     Gson gson = new Gson();
-    response.setContentType("application/json;");
+    response.setContentType("application/json; charset=UTF-8");
```

Comment:
Does this make it work for non-latin languages also? That's awesome!

Reply:
It does, and it also deals with any characters with special accents like ñ!

Reply:
just what i was looking for!